docs/source/contribution/development_for_databricks.md

```diff
@@ -0,0 +1,73 @@
+# Developing Kedro for Databricks
+
+[Databricks](https://www.databricks.com/) is a cloud-based platform for data engineering and data science that many Kedro users deploy their Kedro projects to. Extending and improving the Kedro experience on Databricks is important to a large part of Kedro's user base.
+
+## Introduction
+
+This page describes how to efficiently develop features and fixes for Kedro on Databricks. Using this guide, you will be able to quickly test your locally modified version of Kedro on Databricks as part of a build-and-test development cycle.
```

Comment:
@jmholzer I was initially unsure why this is part of the contribution documentation rather than a revision of the deployment page we already have. I think it's because it's for people who are working on Kedro code itself (not a Kedro project) and need to test the updated Kedro version -- is that correct? I would bring that out a whole lot more and point to the deployment page to say "If you just wanna test your Kedro project, this isn't the page for you, go here instead" kind of thing. Likewise, i'd add something to the existing deployment page to push Kedro contributors over to this one.

Reply:
Correct, this is only for developers looking to improve Kedro 🙂 you're right I should have made this more clear.

I've added the notes you suggested on both the project deployment guide and on this guide (in [64e3afb](https://github.com/kedro-org/kedro/pull/2369/commits/64e3afbc491019c9e962d240ab69984ad0d21910) and [f5e6018](https://github.com/kedro-org/kedro/pull/2369/commits/f5e6018ce0bc66e8c4924e7cb0f9c167d5394c70)).